learn/core_concepts/relevancy.md

```diff
@@ -16,28 +16,29 @@ The order in which they are applied has a significant impact on the search resul
 
 By default, ranking rules are executed in the following order:
 
-**1. Typo**
-Results are sorted by **increasing number of typos**: find documents that match query terms with fewer typos first.
-
-**2. Words**
+**1. Words**
 Results are sorted by **decreasing number of matched query terms** in each matching document: find documents that contain more occurrences of the query terms first.
 
-::: warning
+::: note
+
+Be aware that the `words` rule works from right to left. Therefore, the order of the query string impacts the order of results.
+
+For example, if someone were to search `batman dark knight`, then the `words` rule would rank documents containing all three terms first, documents containing only `batman` and `dark` second, and documents containing only `batman` third. All other documents would be ranked last.
 
-It is now mandatory that all query terms are present in the returned documents. This rule does not impact search results yet. <Badge text="soon" type="warn"/>
+In other words, results containing just `dark` and `knight`, or `batman` and `knight`, would be sorted last, equivalent to documents that contain none of the query terms.
```

Comment:
This cannot be possible, there is no way that the engine return documents that do not contain the first word as the engine tries to remove the words from the end of the query and always requires the documents to contain all of the query words (currently tested). You can remove this sentence :)

Reply:
Thanks for clarifying this @Kerollmops !